test/run-webdriver.js

```diff
@@ -1,14 +1,24 @@
 var WebDriver = require('selenium-webdriver');
+var chromeCapabilities = WebDriver.Capabilities.chrome();
 
 function runWebdriver() {
   var webdriver;
+  // Adding this should fix the weird CI failing issue
+  // https://github.com/SeleniumHQ/selenium/issues/4961
```

Comment:
The linked issue, says only specify `--no-sandbox`, what is the thinking behind adding the other args?

Reply:
When I ssh'd into the container that ran this and added the chromeOption `--no-sandbox` it would still intermittently fail but when I added the options `--headless --disable-gpu` it would consistently pass. https://jakebinstein.com/blog/how-to-set-browser-capabilities-in-webdriverjs-example-headless-mode/